plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/Connection.java

```diff
@@ -88,6 +91,23 @@ public Connection(IClient client, ICredentialsPrompter credentialsPrompter, ISSL
 		this.sslCertificateCallback = sslCertCallback;
 	}
 	
+	@Override
+	public Map<String, Object> getExtendedProperties() {
+		return new HashMap<>(extendedProperties);
+	}
+
+	@Override
+	public void setExtendedProperty(String name, String value) {
```

Comment:
Either the value is an Object or the extendedProperties map is Map<String, String>